styles.css

```diff
@@ -390,7 +391,7 @@ h2 {
   top: 40%;
 }
 
-.pop-up-window #carousel-cancel {
+.pop-up-window .carousel-cancel {
   margin-top: 5%;
   margin-left: 90%;
   margin-bottom: 6%;
```

Comment:
- [ ] Nice work implementing the popup :100: However as requested in the project requirement let stick to the Figma design as much as possible.

### Your design 
![Screenshot from 2023-04-22 11-08-08](https://user-images.githubusercontent.com/91142432/233781277-58f66413-aa7b-4c8c-aa2e-1788c43852a8.png)

### Figma's design 
![Screenshot from 2023-04-22 11-06-05](https://user-images.githubusercontent.com/91142432/233781313-a00c8ee6-3375-4d53-811a-d1f5a9ead234.png)

Reply:
@LionRouge1 hi just to be clear here the issue is the position of the arrows right?

Reply:
@LionRouge1 ????